client/src/sections/landing-page/Benefits.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import Image from 'next/image';
 import type { FC } from 'react';
 
```

Comment:
- to avoid confusion. Please add nalang `Section` at the end.
![image](https://user-images.githubusercontent.com/73863685/228154930-1a7f5d77-102d-4384-84b0-e241c079392c.png)

```
client/src/sections/landing-page/BenefitSection.tsx
```